src/main/java/org/sagebionetworks/web/client/widget/evaluation/EvaluationSubmitter.java

```diff
@@ -443,4 +485,20 @@ public String getSelectedTeamMemberStateHash() {
 	public boolean getIsIndividualSubmission() {
 		return isIndividualSubmission;
 	}
+
+	public void setDigest(DockerCommit commit) {
+		dockerDigest = commit.getDigest();
+	}
+
+	@Override
+	public void onDockerCommitNextButton() {
+		DockerCommit commit = dockerCommitList.getCurrentCommit();
+		if (commit == null) {
+			dockerCommitSynAlert.handleException(new Throwable(NO_COMMITS_SELECTED_MSG));
```

Comment:
just use synAlert.showError(NO_COMMITS_SELECTED_MSG) instead of wrapping with a Throwable.